mcsema/BC/Instruction.cpp

```diff
@@ -364,4 +364,33 @@ llvm::Value *InstructionLifter::LiftAddressOperand(
       inst, block, arg, op);
 }
 
+// Lift the register operand to a value.
+llvm::Value *InstructionLifter::LiftRegisterOperand(
+    remill::Instruction &inst, llvm::BasicBlock *block,
+    llvm::Argument *arg, remill::Operand &op) {
+
+  auto &reg = op.reg;
+
+  // Check if the instruction is referring to the base pointer which
+  // might be accessing stack variable indirectly
+  if (ctx.cfg_inst->stack_var) {
+    if (!std::strcmp(const_cast<const char*>(reg.name.c_str()), "RBP")) {
```

Comment:
Can you move this into a static function, like, `IsFramePointerReg`, which takes in a reference to a `Register`, and then if `gArch->IsAMD64()`, then do the check against `"RBP"`? Also, I don't think the `const_cast` is needed. `const_cast` is typically for removing `const`-ness. @